docs/_guide/03-styling-templates.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Styling templates
+slug: styling-templates
+---
+
+lit-html just renders HTML; you can style HTML created with lit-html like any other HTML, using CSS. How you apply styles to lit-html templates depends on how you're using it—for example, if you're using lit-html inside a component system like LitElement, you can follow the patterns used by that component system.
+
+In most cases you should **not** use lit-html bindings inside a style sheet. It's more efficient to use lit-html bindings to manipulate the `class` and `style` attributes.  
+
+lit-html provides two directives for manipulating an element's `class` and `style` attributes:
+
+*   [`classMap`](template-reference#classmap) sets classes on an element based on the properties of an object.
+*   [`styleMap`](template-reference#stylemap) sets the styles on an element based on a map of style properties and values.
+
+See [classMap](template-reference#classmap) and [styleMap](template-reference#stylemap) in the Template syntax reference.
+
+There are also some special considerations when using lit-html to render into a shadow root.
```

Comment:
I think rather than calling Shadow DOM as special, maybe divide the world into two: you're either going to be using Shadow DOM or not.

If not, you're rendering to the main doc and you should use whatever system you already do. A simple stylesheet or various CSS module / CSS-in-JS systems. If you use those you might need to bind class names (it's common for a CSS module system to have you import class names to put on elements), and that's easy in lit-html. For more information see your CSS libraries docs, etc.

If you are using Shadow DOM (recommended), then... (what you have below)